### DISCUSSION TITLE
Spoofed QuestAppLauncher

### DISCUSSION THREAD

**Discussion by reloxx13** [reloxx13/QuestAppLauncher_Assets#5] [Show and tell]
Its possible to use a spoofed app to launch the QuestAppLauncher from your homescreen.    
     
Youll have to google for `reddit questapplauncher spoofed` and search a little bit.    
    
You´ll find a telegram link (in the sidebar of the subreddit) to a chat group which got a nice bot which tells you where you get the spoofed app.
    
    
Spoofing means the QuestAppLauncher will camouflage as an orginal app which will be on your homescreen.    
If you hit the orginal app, the QuestAppLauncher will start.    
   
    
Again, due copyright i cannot post links here.

**Comment by WilsonBradley** [comment]
Thank you.. I think I'll wait for spoofing the applaucher again since I don't want to get banned.

At this point though, I can't even get the QuestAppLauncher to work in my Quest 2. I run it and it just shows the gear, no icons.
/i tied to follow instructions for placing files, but it's not clear.
